scarlet/blend.py

```diff
@@ -112,11 +116,13 @@ def fit(self, max_iter=200, e_rel=1e-3, min_iter=1, random_skip=0, **alg_kwargs)
 
         return self
 
-    def get_model(self, *parameters):
+    def get_model(self, *parameters, frame=None):
```

Comment:
Consistency. We have (and need) this signature in `Component` and `ComponentTree` because we backpropagate only boxes, not full-frame models. In `Blend` we had overwritten that method, but then it makes a difference if you work with a `Blend` instance or a `ComponentTree` instance, although the former is derived from the latter.